libs/community/tests/integration_tests/vectorstores/test_pinecone.py

```diff
@@ -19,6 +20,35 @@
 dimension = 1536  # dimension of the embeddings
 
 
+@pytest.fixture(scope="module")
```

Comment:
😕  I don't understand how tests were passing without these fixtures being defined, but I'm happy to remove them if there's some magic-fu that I didn't follow (maybe in `cassettes/`? Some pytest config that my grep missed?)

I did notice that the `test_momento_vector_index` does define these fixtures [here](
https://github.com/langchain-ai/langchain/blob/master/libs/community/tests/integration_tests/vectorstores/test_momento_vector_index.py#L29-L43), so not sure why it would be in some test files but not other. Let me know which you prefer and I'll clean it up!